lib/lotus/logger.rb

```diff
@@ -0,0 +1,93 @@
+require 'logger'
+require 'lotus/utils/string'
+
+module Lotus
+  # Lotus default logger
+  #
+  # Implement with the same interface of ruby std ::Logger
+  # Opt out the logdev parameter and always use STDOUT to output the log
+  #
+  # Lotus logger also has the app tag concept, which used to identify
+  # which application the log come from.
+  # Lotus Logger default come with Lotus::Application
+  # and take name of class which inherit Lotus::Application as app_tag
+  #
+  # When stand alone, Lotus Logger try to infer app tag from highest namespace
+  # When has no namespace, Lotus Logger take [Shared] as default app tag
+  #
+  # Examples
```

Comment:
what do you about using `@example` instead of `Example`?

also, I think you should document the way that Lotus app uses this Logger, that is through the singleton way